content/guides/docker-concepts/the-basics/what-is-a-registry.md

```diff
@@ -0,0 +1,135 @@
+
+An image registry is a centralized location for storing and sharing your container images. It can be either public or private. [Docker Hub](https://hub.docker.com) is a public registry that anyone can use and is the default registry. 
+
+While Docker Hub is a popular option, there are many other available container registries available today, including [Amazon Elastic Container Registry(ECR)](https://aws.amazon.com/ecr/), [Azure Container Registry (ACR)](https://azure.microsoft.com/en-in/products/container-registry), and [Google Container Registry (GCR)](https://cloud.google.com/artifact-registry). You can even run your private registry on your local system or inside your organization. For example, Harbor, JFrog Artifactory, GitLab Container registry etc.
+
+### Registry vs repository
+
+While you're working with registries, you might hear the terms **registry** and **repository** as if they're interchangeable. Even though they're related, they're not quite the same thing.
+
+A _registry_ is a centralized location that stores and manages container images, whereas a _repository_ is a collection of related container images within a registry. Think of it as a folder where you organize your images based on projects. Each repository contains one or more container images.
+
+>**Note**
+>
+> You can create one private repository and unlimited public repositories using the free version of Docker Hub. For more information, visit the [Docker Hub subscription page](https://www.docker.com/pricing/).
+
+## Try it now
+
+In this hands-on, you will learn how to build and push a Docker image to the Docker Hub repository.
+
+### Sign up for a free Docker account
+
+1. If you have not done so, [create a Docker account](https://hub.docker.com/signup).
+
+    ![Screenshot of the official Docker Hub page showing the Sign up page](images/dockerhub-signup.webp?border)
+
+    You can use your Google or GitHub account to authenticate.
+
+### Create your first repository
+
+1. Sign in to [Docker Hub](https://hub.docker.com)
+2. Select the **Create repository** button in the top-right corner
+3. Select your namespace (most likely your username) and enter `docker-quickstart` as the repository name.
+
+    ![Screenshot of the Docker Hub page that shows how to create a public repository](images/create-hub-repository.webp?border)
+
+4. Set the visibility to **Public**. 
+5. Select the **Create** button to create the repository.
+
+That's it. You've successfully created your first repository. 🎉
+
+This repository is empty right now. You'll now fix this by pushing an image to it.
+
+### Login with Docker Desktop
+
+1. [Download and install](https://www.docker.com/products/docker-desktop/) Docker Desktop, if not already installed.
+2. In the Docker Desktop GUI, select the **Log in** button in the top-right corner
+
+### Cloning a sample Node.js code
```

Comment:
Nit: Suggest using the same conjugation as the previous task-based sections. 

```suggestion
### Clone sample Node.js code
```
Or `Clone a sample Node.js project`